crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -450,6 +450,10 @@ define_tedge_config! {
             /// Enable auto registration feature
             #[tedge_config(example = "true", default(value = true))]
             auto_register: bool,
+
+            /// Determines if entity state on tedge should be sent to cloud on startup
```

Comment:
I'm not sure this comment is meaningful for an  end-user reading `tedge config list --doc` 

```suggestion
            /// On a clean start, the whole state of the device, services and child-devices is resent to the cloud
```